setup.py

```diff
@@ -109,6 +136,7 @@ def pip_install(package):
                      'coveralls>=1.5',
                      'pytest>=4.6']
 
+# Why are we doing this instead of allowing the environment to do this? We should just use -O3 and -fPIC.
```

Comment:
Could you elaborate? We are checking the environment and if it doesn't specify these flags, we use the default ones. Let me know what you recommend.

Reply:
Well in this case if the user specifys `FCFLAGS="-Wall"` or something completely unrelated to optimisation, we compiled without any optimisation...